packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Internal/ClientUriBuilder.cs

```diff
@@ -56,8 +56,6 @@ public void AppendPath(string value, bool escape)
 
         public void AppendPath(byte[] value, string format, bool escape = true) => AppendPath(TypeFormatters.ConvertToString(value, format), escape);
 
-        public void AppendPath(IEnumerable<string> value, bool escape = true) => AppendPath(TypeFormatters.ConvertToString(value), escape);
```

Comment:
did we mean to remove this?

Reply:
Yes, this was replaced with the generic AppendPathDelimited.